### PR TITLE
Add CerealCodes Problems

### PR DESCRIPTION
**Pull request by jessechoe10**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by jessechoe10** [comment]
More to come later. Don't merge yet.

**Comment by SansPapyrus683** [comment]
aren't there enough combo problems

**Comment by envyaims** [comment]
> aren't there enough combo problems

there can never be enough problems 🤡

**Comment by Aadit-Ambadkar** [comment]
No, there can be enough string problems, bitmask problems, etc. But there can never be enough combo problems.

File: content/6_Advanced/Comb_Sub.problems.json [hunk 1]
```diff
@@ -29,13 +29,13 @@
...
-      "name": "Div 1 D - Miss Punyverse",
-      "url": "https://codeforces.com/contest/1280/problem/D",
+      "uniqueId": "cfgym-103886Q",
+      "name": "Cereal Trees II",
+      "url": "https://codeforces.com/gym/103886/problem/Q",
       "source": "CF",
       "difficulty": "Normal",
       "isStarred": false,
-      "tags": [],
+      "tags": ["Tree DP"],
```
Comment (bqi343) [inline content/6_Advanced/Comb_Sub.problems.json:38]:
```suggestion
      "tags": ["Tree", "DP"],
```

**Review by bqi343 — CHANGES_REQUESTED** [review]
for consistency with other problems

**Comment by Aadit-Ambadkar** [comment]
@jessechoe10 if you are still working on the pr, mark it as a draft.
image

**Comment by dutinmeow** [comment]
A bit off topic but I can create an editorial for CC Vista, if that's ok with you

**Comment by jessechoe10** [comment]
> @jessechoe10 if you are still working on the pr, mark it as a draft. image

@Aadit-Ambadkar can you or @envyaims get a list of our good problems we can add to the guide (module name and difficulty). You can ask the problem authors if they think the problem is good btw.

**Comment by jessechoe10** [comment]
> A bit off topic but I can create an editorial for CC Vista, if that's ok with you

Yeah, can you DM Eric, Envy, or Patrick.

**Comment by jessechoe10** [comment]
> > A bit off topic but I can create an editorial for CC Vista, if that's ok with you
> 
> Yeah, can you DM Eric, Envy, or Patrick?

**Comment by ryanchou-dev** [comment]
lgtm!

**Pull request merged**